next-app/src/routes/password/change/+page.svelte

```diff
@@ -16,6 +16,23 @@
 	$: if (form?.failed) {
 		$error = Error(form.failed)
 	}
+
+	$: new_password_confirm && debounce(() => verify(new_password_confirm, new_password), 400)
+
+	$: submittable = new_password && new_password_confirm && new_password_confirm === new_password
+	$: disabled = ! submittable
+
+	function verify(password_confirm: string, password: string) {
+		password_confirm !== password ? $error = Error('Passwords are not the same') : dismiss()
+	}
+
+	function debounce(fn: () => void, delay: number) {
+		const id = setTimeout(() => {
+			fn()
+
+			clearTimeout(id)
```

Comment:
why the clear after? a timeout is only going to fire once and if you call the debounce function multiple times it'll setup a new scope each time with a new id. Should the id be stored in a variable outside the scope of the function so that we can cancel any pending timeouts whenever this function is called?

Reply:
good catch, thank you, corrected in [6b7cca3](https://github.com/sillsdev/web-languageforge/pull/1719/commits/6b7cca322ac7a50dfcadbbcf1c202d6ccee123f8)

Reply:
I'm going to presume you'd resolve this so I can get it merged.  Please let me know if you would've felt differently.